backend/pkg/database/gorm_repository_migrations.go

```diff
@@ -194,6 +196,35 @@ func (gr *GormRepository) Migrate() error {
 				return nil
 			},
 		},
+		{
+			ID: "20240813222836", // add role to user
+			Migrate: func(tx *gorm.DB) error {
+
+				err := tx.AutoMigrate(
+					&_20240813222836.User{},
+				)
+				if err != nil {
+					return err
+				}
+
+				// set first user to admin
+				// set all other users to user
+				users := []_20240813222836.User{}
+				results := tx.Find(&users)
```

Comment:
is this guaranteed to be sorted in the correct order? IIRC Users have a UUID as the primary key. We should sort by `created_at` and then mark the earliest user as Admin.